src/main/flight/pid.h

```diff
@@ -85,7 +85,6 @@ typedef struct pidProfile_s {
     uint8_t vbatPidCompensation;            // Scale PIDsum to battery voltage
     uint8_t pidAtMinThrottle;               // Disable/Enable pids on zero throttle. Normally even without airmode P and D would be active.
     uint8_t levelAngleLimit;                // Max angle in degrees in level mode
-    uint8_t levelSensitivity;               // Angle mode sensitivity reflected in degrees assuming user using full stick
```

Comment:
Should this stay in to retain backwards compatibility? (At least in theory.)

Reply:
I've increased the PG version: `PG_REGISTER_ARRAY_WITH_RESET_FN(pidProfile_t, MAX_PROFILE_COUNT, pidProfiles, PG_PID_PROFILE, 2)`

Reply:
I see. So for additions to the end of the PG, the version number does not have to be increased, but for changes _inside_ it it has to be?

Reply:
That is correct.